src/sampler/reserve.jl

```diff
@@ -0,0 +1,103 @@
+import Distributions: length, eltype, _rand!
+
+abstract type AbstractReserveSampler end
+
+"""
+    E2ELRReserveScaler{U,F}
+
+Samples reserve requirements following the procedure below:
+
+1. Sample a minimum reserve requirement `MRR` from a uniform distribution `U(lb, ub)`.
+2. Compute the upper bound of reserve requirements for each generator as `rmax = α * (pmax - pmin)`.
+3. Fix the lower bound of reserve requirement per generator to zero.
+
+The parameter `α` is a scaling factor that determines the total reserve requirement
+    as a multiple of the maximum power range of all generators.
```

Comment:
Not sure I understand everything here.

* The `MRR` value is sampled from a uniform distribution --> so far so good, no issue.
* Items 2) and 3) then give `rmax = α * (pmax - pmin)` (I assume this is per generator?), and `rmin = 0` --> OK
* The last paragraph then says that  `α` "determines the total reserve requirement": isn't that MRR, which was supposed to be uniform?

Looking at the sampling code below, I think the docstring may need to be updated.